packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -61,6 +64,10 @@ const _Dropdown = ({
   const [hasFooterAction, setHasFooterAction] = React.useState(false);
   const [hasLabelOnLeft, setHasLabelOnLeft] = React.useState(false);
   const [isKeydownPressed, setIsKeydownPressed] = React.useState(false);
+  const [changeCallbackTriggerer, setChangeCallbackTriggerer] = React.useState<
```

Comment:
It can be used in dependency array to know when the change has happened. The tricky part for us is that our `onChange` is on `SelectInput` component but that onChange is supposed to be triggered when ActionList item has some change. So this is the way to tell SelectInput that something has changed.

Reply:
Added jsdoc description